exporter/loadbalancingexporter/trace_exporter_test.go

```diff
@@ -157,6 +158,40 @@ func TestConsumeTraces(t *testing.T) {
 	assert.Nil(t, res)
 }
 
+func TestConsumeTracesServiceBased(t *testing.T) {
```

Comment:
Can you also have a test where the same traceID is used, but different service names are used? This would ensure that service names are the decisive factor in routing the spans.